test/config_test/example_configs_test.cc

```diff
@@ -17,10 +17,11 @@ TEST(ExampleConfigsTest, All) {
 
 #ifdef __APPLE__
   // freebind/freebind.yaml is not supported on OS X and disabled via Bazel.
-  EXPECT_EQ(28UL, ConfigTest::run(directory));
+  EXPECT_EQ(26UL, ConfigTest::run(directory));
```

Comment:
google_com_proxy.json is one of them, which one is the other one?

Reply:
It's google_com_proxy.json and google_com_proxy.yaml. left only google_com_proxy.v2.yaml